src/NServiceBus.Core.Tests/Persistence/InMemory/When_persisting_a_saga_with_no_defined_unique_property.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace NServiceBus.Core.Tests.Persistence.InMemory
+{
+    using System;
+    using System.Threading.Tasks;
+    using Extensibility;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class When_persisting_a_saga_with_no_defined_unique_property
```

Comment:
This is the same tests to verify that things work when there is no correlation property configured because a custom finder is being used.